fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -300,6 +300,7 @@ classifier
 
 showDatabasesStatement
     : SHOW DATABASES ((FROM | IN) catalog=qualifiedName)? ((LIKE pattern=string) | (WHERE expression))?
+    | SHOW SCHEMAS ((LIKE pattern=string) | (WHERE expression))?
```

Comment:
Why not SHOW (DATABASES|SCHEMAS) ? then all the syntax can be supported for schemas.

Reply:
Mysql does not normally support the syntax of SHOW SCHEMAS (FROM/IN), and SHOW DATABASE support should be a consideration of compatibility. Does SHOW SCHEMAS will support (FROM/IN) syntax is OK?

Reply:
Let's keep compatible with MySQL firstly.